### PR TITLE
Enrichment protocol update. Fixes #925

### LINKED ISSUE #925: Change `enrichment_method ` to `method`, `min_size_selected` to `minimum_size`, `max_size_selected` to `maximum_size`
**For which schema is a change/update being suggested?**

I would like to request an update to the `enrichment_protocol.json` schema.

**What should the change/update be?**

- [ ] change `enrichment_method ` to `method`  to remove redundancy ("enrichment")
- [ ] change `min_size_selected` to `minimum_size`
- [ ] change `max_size_selected` to `maximum_size`

**What new field(s) need to be changed/added?**

Above.

**Why is the change requested?**

Adhere to Style Guide.

### PR DESCRIPTION
**Pull request by ESapenaVentura**
### Release notes

For `enrichment_protocol.json` schema:
- Changed `enrichment_method` to `method`
- Changed `min_size_selected` to `minimum_size`
- Changed `max_size_selected` to `maximum_size`
 

### Reviews requested

- Need 2 Reviewers to approve because this is a major update
- Last Reviewer to review/approve, merge changes by 16/04/2019

### CODE REVIEW

**Comment by malloryfreeberg** [comment]
@zperova if you want to review this first, I'll do the merge after you

**Comment by zperova** [comment]
I thought you trust me, @malloryfreeberg? I was waiting for you to do it first.

**Comment by malloryfreeberg** [comment]
I do trust you! I just haven't done one in a while and I miss it :(

File: json_schema/type/protocol/biomaterial_collection/enrichment_protocol.json [hunk 1]
```diff
@@ -54,15 +54,15 @@
             "user_friendly": "Markers",
             "guidelines": "Format as a space-delimited list of markers with +/- where + means enriched for and - means enriched against."
         },
-        "min_size_selected": {
+        "minimum_size": {
             "description": "Minimum cell or organelle size passing selection, in microns.",
             "type": "number",
             "maximum": 10000,
```
Comment (zperova) [inline json_schema/type/protocol/biomaterial_collection/enrichment_protocol.json]: is there a particular reason for "maximum" and "minimum" values to be in both "minimum_size" and "maximum_size" and not just in the corresponding field?

**Comment by zperova** [comment]
Requesting @daniwelter to review as this has the property_migrations.json

**Pull request merged**